src/platform/BUILD.gn

```diff
@@ -68,6 +68,12 @@ if (chip_device_platform != "none" && chip_device_platform != "external") {
 
     # Define the default number of ip addresses to discover
     chip_max_discovered_ip_addresses = 5
+
+    # Total number of UDC messages to be sent
+    chip_total_udc_message_count = 5
+
+    # Delay between sending repeated UDC messages (in ms)
+    chip_udc_message_delay_ms = 100
```

Comment:
Is this per spec to be 100 ms or is it expected to be configurable? Like the idea should we just hard code it as a macro or a const int in code or should it be configurable and updatable via a config file?

Reply:
The spec requires waiting for at least 100ms, but some app / device can choose to wait for longer if it wants to. For those apps / devices, it is useful to have it as a gn config.